tests/test_e2e_01_kytos_startup.py

```diff
@@ -32,9 +35,14 @@ def setup_class(cls):
         cls.net.start()
         cls.net.wait_switches_connect()
         # rotate logfile (copy/truncate strategy)
-        logfile = '/var/log/syslog'
-        shutil.copy(logfile, logfile + '-' + time.strftime("%Y%m%d%H%M%S"))
-        open(logfile, 'w').close()
+        try:
+            cls.logfile = '/var/log/syslog'
+            if not os.path.exists(cls.logfile):
+                raise Exception("/var/log/syslog does not exist")
+            shutil.copy(cls.logfile, cls.logfile + '-' + time.strftime("%Y%m%d%H%M%S"))
+            open(cls.logfile, 'w').close()
+        except Exception as e:
```

Comment:
We do have a few raised broad exception being raised and handled, but this can be simplified, could you refactor this as, that way the intention becomes clearer that it's just ignoring the file if it's not found, leading to fewer surprises in the future too as this part might get refactored:

```
try:
    logfile = '/var/log/syslog'
    shutil.copy(logfile, logfile + '-' + time.strftime("%Y%m%d%H%M%S"))
    open(logfile, 'w').close()
except FileNotFoundError:
    pass
```